main/src/main/java/org/mobilitydata/gtfsvalidator/validator/BookingRulesEntityValidator.java

```diff
@@ -144,4 +153,34 @@ static class ForbiddenSameDayBookingFieldValueNotice extends ValidationNotice {
       this.fieldNames = String.join(", ", forbiddenFields);
     }
   }
+
+  /**
```

Comment:
Is this the info that you want highlighted in rules.html? 
Usually there is a short description that will be displayed in bold, followed by a longer explanation.  Your description seems a bit long to be the short description.

Reply:
Ok, shortened it to   /** prior notice last day should not be greater than the start day in booking_rules.txt. */